### PR TITLE
Add support for the Steam Web API

### LINKED ISSUE #186: Add support for the Steam Web API
**Describe the solution you'd like**
It makes sense for us to integrate the [Steam Web API](https://partner.steamgames.com/doc/webapi_overview) to get access to the [ISteamNews Interface](https://partner.steamgames.com/doc/webapi/ISteamNews). This way we can query blog posts on Steam directly, instead of having to parse the RSS feed (this seems to have a big delay).

### PR DESCRIPTION
**Pull request by TimJentzsch**
**Description**:
Allows the Steam Web API to act as provider, more specifically the ISteamNews Interface.
Also fixes the `notifyGameSubs` and `notifyAllSubs` methods to properly obtain the channels, so that the appropriate labels are used.
Closes #186.

**Checklist**:
- [x] Tested the bot on both clients with a few commands
- [ ] Updated the [`README`](README.md) if necessary
- [ ] Updated the version string in the [`package.json`](package.json) if necessary

### CODE REVIEW

**Comment by TimJentzsch** [comment]
Still to do: Fix the formatting.
The formatting of the ISteamNews interface is provided via Steam tags. However this formatting is lost at some point, resulting in plain text posts:
No formatting on Steam update posts

File: package.json [hunk 1]
```diff
@@ -56,7 +56,10 @@
     "typescript": "3.8.3"
   },
   "dependencies": {
+    "@types/axios": "^0.14.0",
     "@types/cheerio": "0.22.16",
+    "@types/circular-json": "^0.4.0",
```
Comment (zachkont) [inline package.json:61]: please change all versions to not include a caret

File: package.json [hunk 2]
```diff
@@ -65,6 +68,7 @@
     "@types/showdown": "1.9.3",
     "@types/ws": "7.2.3",
     "@types/xml2js": "0.4.5",
+    "axios": "^0.19.2",
```
Comment (zachkont) [inline package.json:71]: this one too

File: package.json (hunk 1 above)
Comment (TimJentzsch) [reply package.json]: Fixed.

File: package.json (hunk 2 above)
Comment (TimJentzsch) [reply package.json]: Fixed.

**Pull request merged**